### ISSUE TITLE
Move normalization/denormalization onto the server

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/DeepClean#22]
The channel-wise normalization and denormalization of the noise estimate right now are handled on the client side via pickled `Op` objects. However, this adds another two files for client pipelines to keep track of, and as the discussion in #20 outlines, might create some convoluted logic when dealing with channels which have dropped out either during training or at inference time.

Things might be simplified if during model export we created two `torch.nn.Module`s which apply these steps before and after the model, and ship them to the server as a single model. Something like

```python
import pickle
import torch
from deepclean.architectures import DeepCleanAE


def Normalizer(torch.nn.Module):
    def __init__(self, pipeline_pkl: str, inverse: bool = False):
        super().__init__()

        with open(pipeline_pkl, "rb") as f:
            scaler = pickle.load(f).ops[0]
            self.mean = torch.Tensor(processor.mean, requires_grad=False)
            self.std = torch.Tensor(processor.std, requires_grad=False)
        self.inverse = inverse

    def forward(self, x):
        if inverse:
            return self.std * x + self.mean
        return (x - self.mean) / self.std


num_chanels = 21
normalizer = Normalizer("/path/to/witness_pipeline.pkl")
denormalizer = Normalizer("/path/to/strain_pipeline.pkl", inverse=True)
deepclean = DeepCleanAE(num_channels)
model_to_export = torch.nn.Sequential([normalizer, deepclean, denormalizer])
do_regular_export(model_to_export)
```
which will ship all these steps off as a single model on the server. The snapshotter obviously won't be affected by this, and since its just a linear transformation the aggregator won't either, so nothing else on the server should need to change.
